graphene_sqlalchemy/tests/test_converter.py

```diff
@@ -142,18 +134,26 @@ def test_should_label_convert_int():
     assert isinstance(graphene_type, graphene.Int)
 
 
-def test_should_choice_convert_enum():
+def test_should_choice_convert_list():
```

Comment:
The name is a bit confusing. It reads like "a choice should be converted to a list type". Something like `test_should_list_choice_convert_enum" would be clearer.